packages/jest-jasmine2/src/index.js

```diff
@@ -69,7 +69,49 @@ function jasmine2(config, environment, moduleLoader, testPath) {
       moduleLoader.requireModule(null, config.setupTestFrameworkScriptFile);
     }
   });
+
+  const hasIterator = object => !!(object !== null && object[Symbol.iterator]);
```

Comment:
this function gets called only for typeof === 'object', so undefined is excluded, I guess it would be better to change this anyway so that if anyone uses it in the future doesn't get tricked by it